R/pkgdown.R

```diff
@@ -28,6 +28,11 @@ use_pkgdown <- function(config_file = "_pkgdown.yml", destdir = "docs") {
   use_build_ignore("pkgdown")
```

Comment:
Should be (build + git) ignoring `docs` here too? (I think so, given that the github actions flow is quite straighforward)

Reply:
We do that already:

https://github.com/r-lib/usethis/blob/94a389144d661d392ccf9636b209e04285704b62/R/pkgdown.R#L27-L29